Libplanet.Tools/ConsolePasswordReader.cs

```diff
@@ -0,0 +1,46 @@
+using System;
+using System.Text;
+
+namespace Libplanet.Tools
+{
+    public class ConsolePasswordReader
```

Comment:
Shouldn't this new class be used elsewhere (to replace `GetPass`)?

Reply:
In currently, GetPass is only used on Libplanet.Tools. I think it'll be fine to be here.